### PR TITLE
Rust 1.79.0

### PR DESCRIPTION
**Pull request by navi-desu**
tests pass, missing a build test with latest firefox (127.0) will run in the morning tomorrow.

also added the limits to <1.78 for spidermonkey too, as i forgot it on the 1.78 bump.

---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by juippis** [comment]
Spidermonkey **should** be fine, rust-simd is disabled there now.
91e98551c62d8d9987c8403c0c76309d4ed0f4b4
1970d5f0ba0511308f7ec41ed2fd4c6df58314a6

**Comment by navi-desu** [comment]
> Spidermonkey **should** be fine, rust-simd is disabled there now.

oh, i didn't notice, tho, simd is only disabled on 115.11 and 115.12, both currently ~amd64. the two currently stable versions 115.9.1 and 115.10.0 still fail to build with >=1.78 (this is why i didn't notice)

**Comment by navi-desu** [comment]
i removed the limit for 115.11, but kept it for the two other latest stable versions, 115.10 and 102.15 (slot :102)

**Comment by thesamesam** [comment]
Thank you!

File: dev-lang/spidermonkey/spidermonkey-115.10.0.ebuild [hunk 1]
```diff
@@ -78,7 +78,7 @@ BDEPEND="${PYTHON_DEPS}
 			clang? (
 				sys-devel/lld:18
 				sys-devel/clang:18
-				virtual/rust:0/llvm-18
+				<virtual/rust-1.78:0/llvm-18
```
Comment (thesamesam) [inline dev-lang/spidermonkey/spidermonkey-115.10.0.ebuild:81]: I don't think anything will satisfy this for 1.78 in the 0/llvm-18 subslot but I don't think it's _wrong_ either.

**Comment by thesamesam** [comment]
I pushed 686b9338a7a42dbb4c632e4ac228eaf9e01c003b as a followup too.

**Pull request closed without merging**